medical/models/medical_patient.py

```diff
@@ -33,73 +32,90 @@ class MedicalPatient(models.Model):
     _description = 'Medical Patient'
     _inherits = {'res.partner': 'partner_id', }
 
-    @api.one
-    def _compute_age(self):
+    age = fields.Char(
+        compute='_compute_age',
+    )
+    identification_code = fields.Char(
+        string='Internal Identification',
+        help=_(
```

Comment:
@pedrobaeza Quick question if you don't mind - Is the `_` method also implicit when raising exceptions, or do we need to call it there? e.g. `raise ValidationError(_("My Error"))`